### PR TITLE
moneropedia add p2pool sidechain merge-mining

### LINKED ISSUE #1845: Add Moneropedia entry P2Pool
After the blog post is out (https://github.com/monero-project/monero-site/pull/1835), we should add a Moneropedia entry for P2Pool.

### PR DESCRIPTION
**Pull request by QuickBASIC**
First PR for a Moneropedia page after translation changes, so let me know if it's missing anything.

Adds pages for P2Pool, sidechain, and merge mining. A lot of the text is borrowed from announcement blog post but I attempted to adjust for appropriate tone. Feedback is appreciated.

Resolves #1845

### CODE REVIEW

File: _i18n/ar/resources/moneropedia/p2pool.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+entry: "P2Pool"
+terms: ["P2Pool", "monero-p2pool"]
+summary: "Peer to peer mining pool for Monero"
+---
+
+{% include disclaimer.html translated="no" translationOutdated="no" %}
+
+### The Basics
+
+Monero P2Pool is a peer-to-peer Monero @mining pool developed by SChernykh (also known as sech1).
```
Comment (erciccione) [inline _i18n/ar/resources/moneropedia/p2pool.md:11]: If there aren't two spaces after a string, markdown won't add a newline. I would suggest to avoid the double space or translators will have to deal with multiple `\n` in the strings. Better either add a newline or everything in the same string.

File: _i18n/ar/resources/moneropedia/merge-mining.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+Merge mining is the process of @mining two or more @blockchains at the same time. Generally there is a parent @blockchain and an auxillary blockchain both of which use the same proof of work algorithm. In most cases the auxillary chain will have a lower difficulty than the parent block chain.
+
+Merge mining can be used for securing a lower hashrate blockchain or developing a @sidechain that does something different than the primary chain.
+Some examples that currently exist or are in development are: a blockchain game (TownForge), digital assets platform (Tari) or a decentralized mining pool (@P2Pool).
+
+#### External Links
+
+- [Tari - What is Tari](https://www.tari.com/#what-is-tari)
+- [Townforge -About](https://townforge.net/about)
+- [P2Pool GitHub](https://townforge.net/about)
```
Comment (erciccione) [inline _i18n/ar/resources/moneropedia/merge-mining.md:20]: I don't think it's necessary to have a dedicated section for external links. Maybe we could add them directly in the paragraph where we mention them? We also don't need to add a link to p2pool website if we link to the Moneropedia entry.

```suggestion
Merge mining can be used for securing a lower hashrate blockchain or developing a @sidechain that does something different than the primary chain.
Some examples that currently exist or are in development are: a blockchain game ([TownForge](https://townforge.net/about)), digital assets platform ([Tari](https://www.tari.com/#what-is-tari)) or a decentralized mining pool (@P2Pool).
```

File: _i18n/ar/resources/moneropedia/p2pool.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+Monero P2Pool is written from scratch in C++. It uses the high-performance libuv library so each node is able to handle thousands of concurrent miner connections.
+There are 2160 blocks in the PPLNS window with a block time of 10 seconds (approximately 6 hours).
+The number of blocks was chosen so that the minimum payout would be approximately 0.0004 XMR (this amount was considered high enough that it could be transferred with minimal fees).
+Each individual miner payout takes only 38 bytes on the Monero blockchain.
+
+### External Links
+
+- [P2Pool source](https://github.com/SChernykh/p2pool)
+- [P2Pool releases](https://github.com/SChernykh/p2pool/releases)
```
Comment (erciccione) [inline _i18n/ar/resources/moneropedia/p2pool.md:40]: I would add only the first link. People will be able to easily find releases on the repository. No need to add a dedicated link.

File: _i18n/ar/resources/moneropedia/sidechain.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+summary: "a blockchain that is merge mined with a primary blockchain"
+---
+
+{% include disclaimer.html translated="no" translationOutdated="no" %}
+
+### The Basics
+
+A sidechain (also known as an auxillary chain) is a @blockchain that is @merge-mined with Monero or other cryptocurrency. In most cases the sidechain will have a lower difficulty than the primary blockchain.
+
+A pegged sidechain is a sidechain that allows you to transfer assets between different blockchains by pegging the value of one asset to another on an alternate blockchain. Currently there are no pegged sidechains that exist for Monero.
```
Comment (erciccione) [inline _i18n/ar/resources/moneropedia/sidechain.md:13]: I would rephrase this or people won't understand why a pegged blockchain is mentioned. Something like

```suggestion
A sidechain that that allows you to transfer assets between different blockchains is called a "pegged blockchain". It works by pegging the value of one asset to another on an alternate blockchain. Currently there are no pegged sidechains that exist for Monero.
```

I would also add between brackets a short explanation of what "pegging" means.

**Review by erciccione — CHANGES_REQUESTED** [review]
The commit description isn't clear about the fact that you are adding three different moneropedia entries. I also think would be better to have three separated commits for each entry to make things more clear.

Also, punctuation needs to be revisited, but i leave that review to native speakers for now.

File: _i18n/ar/resources/moneropedia/p2pool.md (hunk 1 above)
Comment (QuickBASIC) [reply _i18n/ar/resources/moneropedia/p2pool.md:11]: I think you're referring to the /n in the middle of the paragraph here. Per @hyc on my last PR, long lines are harder for reviewers to give feedback on. If keeping the paragraph on one line is preferred I can do it that way. In this case line 11 and line 12 are the same paragraph, and I'd lean towards making the translators lives easier.

Comment (QuickBASIC) [reply _i18n/ar/resources/moneropedia/p2pool.md:11]: I'm re-reading this 8 hours later I'm still not sure I'm tracking correctly. I'm assuming that you mean don't do multiple lines inside of paragraphs. I intended that the single /n would render all in same paragraph and did /n/n for paragraph breaks, but I'm understanding that makes translation harder. Let me know after I make my next commit if I've misunderstood.

File: _i18n/ar/resources/moneropedia/sidechain.md (hunk 4 above)
Comment (QuickBASIC) [reply _i18n/ar/resources/moneropedia/sidechain.md:13]: > a short explanation of what "pegging" means.

Ack. We don't want people having to search the internet for that.

File: _i18n/ar/resources/moneropedia/p2pool.md (hunk 1 above)
Comment (erciccione) [reply _i18n/ar/resources/moneropedia/p2pool.md]: Now looks good :)

File: _i18n/ar/resources/moneropedia/sidechain.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+summary: "a blockchain that is merge mined with a primary blockchain"
+---
+
+{% include disclaimer.html translated="no" translationOutdated="no" %}
+
+### The Basics
+
+A sidechain (also known as an auxillary chain) is a @blockchain that is @merge-mined with Monero or other cryptocurrency. In most cases the sidechain will have a lower difficulty than the primary blockchain.
+
+A sidechain that that allows you to transfer assets between different blockchains is called a "pegged blockchain". It works by setting the value of one asset equal to the value of another asset on an alternate blockchain, otherwise known as pegging. Currently there are no pegged sidechains that exist for Monero.
```
Comment (erciccione) [inline _i18n/ar/resources/moneropedia/sidechain.md:13]:
```suggestion
A sidechain that allows you to transfer assets between different blockchains is called a "pegged blockchain". It works by setting the value of one asset equal to the value of another asset on an alternate blockchain, otherwise known as pegging. Currently there are no pegged sidechains that exist for Monero.
```

File: _i18n/ar/resources/moneropedia/merge-mining.md [hunk 6]
```diff
@@ -11,10 +11,4 @@ summary: "the process of mining two or more blockchains at the same time "
...
 
 Merge mining can be used for securing a lower hashrate blockchain or developing a @sidechain that does something different than the primary chain.
-Some examples that currently exist or are in development are: a blockchain game (TownForge), digital assets platform (Tari) or a decentralized mining pool (@P2Pool).
-
-#### External Links
-
-- [Tari - What is Tari](https://www.tari.com/#what-is-tari)
-- [Townforge -About](https://townforge.net/about)
-- [P2Pool GitHub](https://townforge.net/about)
+Some examples that currently exist or are in development are: a blockchain game [TownForge](https://townforge.net/about), digital assets platform [Tari](https://www.tari.com/#what-is-tari) or a decentralized mining pool (@P2Pool).
```
Comment (SamsungGalaxyPlayer) [inline _i18n/ar/resources/moneropedia/merge-mining.md:14]: `, and a decentralized mining pool`

Comment (QuickBASIC) [reply _i18n/ar/resources/moneropedia/merge-mining.md]: Ack.

Comment (hyc) [reply _i18n/ar/resources/moneropedia/merge-mining.md]: Frankly it looks cleaner with the bulleted list of links. Only problem is the P2Pool GitHub link is a copy/paste of the townforge link.

**Comment by erciccione** [comment]
Should this be closed since it's included in #2074?

**Comment by plowsof** [comment]
yep @erciccione
